virt/virt-2-4-release-notes.adoc

```diff
@@ -132,20 +127,11 @@ than a unique one from a pool. Although rare, randomly assigned MAC addresses ca
 from the CLI by running the following `oc` patch command:
 +
 ----
-$ export TARGET_NAMESPACE=openshift-cnv CNV_CHANNEL=2.3 && oc patch -n "${TARGET_NAMESPACE}" $(oc get subscription -n ${TARGET_NAMESPACE} --no-headers -o name) --type='json' -p='[{"op": "replace", "path": "/spec/channel", "value":"'${CNV_CHANNEL}'"}, {"op": "replace", "path": "/spec/installPlanApproval", "value":"Automatic"}]'
+$ export TARGET_NAMESPACE=openshift-cnv CNV_CHANNEL=2.4 && oc patch -n "${TARGET_NAMESPACE}" $(oc get subscription -n ${TARGET_NAMESPACE} --no-headers -o name) --type='json' -p='[{"op": "replace", "path": "/spec/channel", "value":"'${CNV_CHANNEL}'"}, {"op": "replace", "path": "/spec/installPlanApproval", "value":"Automatic"}]'
 ----
 +
-This command points your subscription to upgrade channel `2.3` and enables automatic updates.
-
-* If you have {VirtProductName} 2.1.0 deployed, you must first upgrade {VirtProductName}.
-//Deleted link:https://bugzilla.redhat.com/show_bug.cgi?id=1817394[*BZ#1817394*] - It was closed.
-//Don't remove: this BZ is probably true fo all 2.x releases.
-** The `masquerade` binding method for virtual machines cannot be used in clusters with RHEL 7 compute nodes.
-
-** As a workaround, select *SATA* as the CD-ROM interface when you create virtual machines and virtual machine templates.
+This command points your subscription to upgrade channel `2.4` and enables automatic updates.
 
-// For 2.3: Add new Known Issues above this line (so that we don't mix the new with the old/possibly no longer irrelevant ones)
-// The following 7 bugs are from the 2.2 release notes but from their bugs they still seem to be relevant for 2.3:
```

Comment:
Comments from 2.3 that are no longer required (and just seemed to make sense removing since I was removing a block of text)